yt/utilities/lib/misc_utilities.pyx

```diff
@@ -35,6 +35,9 @@ cdef extern from "platform_dep.h":
     # NOTE that size_t might not be int
     void *alloca(int)
 
+from cython.parallel import prange, parallel, threadid
+cimport openmp 
```

Comment:
`openmp` is never used below

Reply:
and in fact, it *has* to be removed, since `cimport openmp` is euivalent to `#include "omp.h"`, which will fail on systems without OpenMP support (e.g. MacOS).